tests/ops/functions/test_equal.py

```diff
@@ -1071,7 +1072,7 @@ def jax_assertion_func(x, other_tensor):
 
         # TF and Autograd
         # ------------------
-        with tf.GradientTape() as tape:
+        with tf.GradientTape() as _:
```

Comment:
```suggestion
        with tf.GradientTape():
```
you don't have to make the changes here, but just as an FYI this would be fine too :D